Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM golang:1.15 as builder
```

Comment:
Shouldn't this file be in a different PR?

Reply:
I'll remove Dockerfile, was just curious if you need it.